2022-02-07 07:51:28,811 INFO    MainThread:29689 [wandb_setup.py:_flush():71] setting env: {}
2022-02-07 07:51:28,811 INFO    MainThread:29689 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-07 07:51:28,812 INFO    MainThread:29689 [wandb_init.py:_log_setup():371] Logging user logs to /home/sohui/code/DTC/code/wandb/run-20220207_075128-1kiomi3p/logs/debug.log
2022-02-07 07:51:28,812 INFO    MainThread:29689 [wandb_init.py:_log_setup():372] Logging internal logs to /home/sohui/code/DTC/code/wandb/run-20220207_075128-1kiomi3p/logs/debug-internal.log
2022-02-07 07:51:28,813 INFO    MainThread:29689 [wandb_init.py:init():404] calling init triggers
2022-02-07 07:51:28,813 INFO    MainThread:29689 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {'mdoel': 'DTC', 'server': 57, 'image size': '112,112,80'}
2022-02-07 07:51:28,813 INFO    MainThread:29689 [wandb_init.py:init():449] starting backend
2022-02-07 07:51:28,813 INFO    MainThread:29689 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-07 07:51:28,828 INFO    MainThread:29689 [backend.py:ensure_launched():199] starting backend process...
2022-02-07 07:51:28,835 INFO    MainThread:29689 [backend.py:ensure_launched():204] started backend process with pid: 29773
2022-02-07 07:51:28,837 INFO    MainThread:29689 [wandb_init.py:init():458] backend started and connected
2022-02-07 07:51:28,839 INFO    MainThread:29689 [wandb_init.py:init():519] updated telemetry
2022-02-07 07:51:28,841 INFO    MainThread:29689 [wandb_init.py:init():549] communicating current version
2022-02-07 07:51:29,703 INFO    MainThread:29689 [wandb_init.py:init():554] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-07 07:51:29,704 INFO    MainThread:29689 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-02-07 07:51:29,950 INFO    MainThread:29689 [wandb_init.py:init():592] starting run threads in backend
2022-02-07 07:51:30,184 INFO    MainThread:29689 [wandb_run.py:_console_start():1816] atexit reg
2022-02-07 07:51:30,185 INFO    MainThread:29689 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2022-02-07 07:51:30,186 INFO    MainThread:29689 [wandb_run.py:_redirect():1695] Redirecting console.
2022-02-07 07:51:30,191 INFO    MainThread:29689 [wandb_run.py:_redirect():1751] Redirects installed.
2022-02-07 07:51:30,191 INFO    MainThread:29689 [wandb_init.py:init():619] run started, returning control to user process
2022-02-07 07:53:21,211 INFO    MainThread:29689 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 255
2022-02-07 07:53:21,211 INFO    MainThread:29689 [wandb_run.py:_restore():1758] restore
2022-02-07 07:53:23,938 INFO    MainThread:29689 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 576
  total_bytes: 576
}

